js/apps/system/_admin/aardvark/APP/frontend/js/views/settingsView.js

```diff
@@ -146,7 +146,27 @@
 
           var buttons = [];
           var tableContent = [];
-
+          var extendedNames = window.frontendConfig.extendedNames;
+          var traditionalNameValidation = [{
+            rule: Joi.string().regex(/^[a-zA-Z]/),
```

Comment:
This is a repetition of the validation rules used in collectionsView.js. 
Instead of repeating the rules here and having to change them in multiple places, they should be put into functions and shared between call sites.